src/chart/options/options.cpp

```diff
@@ -115,6 +115,9 @@ void Options::drilldownTo(const Options &other)
 	for (auto &&dim : other.getChannels().getDimensions())
 		if (!getChannels().isSeriesUsed(dim))
 			stackChannel.addSeries(dim);
+	if (stackChannel.isDimension()
+	    && geometry == ShapeType::rectangle)
+		this->align = Base::Align::Type::stretch;
```

Comment:
It smells a bit "workaroundy". Stretch does the work, but it's purpose is complety different. 
I feel like that instead, we would need an `bool spacing` member in `Channel` which gives controls to manually swich on/off the spacing.

Reply:
I noted. I'll not rewrite it in this PR but handle it at the axis refactor.